rosco-web/config/rosco.yml

```diff
@@ -32,6 +32,8 @@ executionStatusToBakeResults:
 # It is safe to leave this out (or blank) if you do not need to configure your own apt repository.
 # The following commented-out line is an example of what a valid entry looks like.
 # debianRepository: http://dl.bintray.com/spinnaker/ospackages ./
+# yumRepository: https://https://jfrog.bintray.com/...
```

Comment:
The comment refers to an apt repository. Shouldn't this be updated now that there are two types of supported repositories?